docs/changelogs.md

```diff
@@ -0,0 +1,410 @@
+# Guide to Changelogs
+
+Changelogs are invaluable for capturing and communicating releases made to projects over time. Maintaining changelogs effectively is undeniably a bit of an art, but a lot of science goes into it as well, and this document aims to guide engineers accordingly.
+
+## tl;dr
+
+- Every MetaMask project for which new versions are distributed publicly should have a changelog file, and it should be called `CHANGELOG.md`.
+- A changelog should be written primarily for consumers of the project, and as such, it should be valuable to them at all times.
```

Comment:
Perhaps worth reminding people that even if no one else uses the package now, someone may want to in the future? i.e. even if a team that produces a package are their own consumers, it's worth thinking about other teams.